src/reducers/TemperatureSyncReducer.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2020
+ */
+
+import { TEMPERATURE_SYNC_ACTIONS } from '../actions/TemperatureSyncActions';
+
+const TEMPERATURE_SYNC_STATES = {
+  SCANNING: 'SCANNING',
+  SCAN_ERROR: 'SCAN_ERROR',
+};
+
+const initialState = () => ({
+  progress: 0,
+  total: 0,
+  syncState: null,
+  isSyncing: false,
+});
```

Comment:
really basic state at the moment. The progress/total should just be for downloading logs, and the `syncState` can possibly help the UI determine what text to display, but will see if that's enough. Trying to have the reducer state reasonably generic where you could chuck it into another app quite easily